Weather App/UI/SettingsVC/SettingsViewController.swift

```diff
@@ -8,16 +8,18 @@
 import UIKit
 
 class SettingsViewController: UIViewController {
-    private lazy var settingsView = SettingsView()
+    lazy var settingsView = SettingsView()
     var viewModel: SettingsViewModel!
     
     //MARK: LifeCycle
     override func loadView() {
         self.view = settingsView
+        viewModel.onEnter?()
```

Comment:
`LoadView` method is used ONLY for loading the main view. Why is this callback called?

Reply:
The callback got removed completely. It ended up being useless.